services/webhook/tests/test_living_hunt.py

```
@@ -0,0 +1,76 @@
+    assert title.startswith("Living Hunt abc12345..def67890 - ")
+
+
+def test_fetch_pr_diff_scope_reports_full_fallback(monkeypatch):
+    responses = [
+        httpx.Response(422, request=httpx.Request("GET", "https://compare")),
+        httpx.Response(
+            200, text="full diff",
+            request=httpx.Request("GET", "https://pull"),
+        ),
+    ]
+    monkeypatch.setattr(cr_dispatch.httpx, "get", lambda *a, **k: responses.pop(0))
```

Ruff (0.15.21)

[warning] Unused lambda argument: `a`

(ARG005)